<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_Equipements</class>
 <widget class="QMainWindow" name="MainWindow_Equipements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>872</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Cambria</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/myappico.png</normaloff>:/myappico.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1211</width>
      <height>831</height>
     </rect>
    </property>
    <property name="title">
     <string>Gestion Equipements</string>
    </property>
    <widget class="QLabel" name="nom_gestion">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>451</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QLabel {
   color:#ffb001;
}
</string>
     </property>
     <property name="text">
      <string>Gestion Equipements</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="afficher">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>120</y>
       <width>621</width>
       <height>661</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    font: 12px;
 background-color:transparent;
}


QLabel {
    font-weight: bold;
    padding: 1px;
}
QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 10px;
}
QComboBox {
    border: 2px solid gray;
     border-radius: 5px;
    padding: 5px;
}

QPushButton {
    background-color: rgb(58, 195, 197);
    color: white;
    border-radius: 5px;
    padding: 5px;
}

QLineEdit:focus, QPushButton:focus {
    border: 2px solid #4CAF50;
}

</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="afficher_label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
   color:#ffb001;
}</string>
      </property>
      <property name="text">
       <string>Liste Equipements</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rechercher">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Icones/Icones/rechercher.png</normaloff>:/Icones/Icones/rechercher.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_trier">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>Lancer Tri</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPDF">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>620</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
    background-color: rgb(58, 195, 197);
    color: white;
    border-radius: 5px;
    padding: 5px;
}

QLineEdit:focus, QPushButton:focus {
    border: 2px solid #4CAF50;
}

</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Icones/Icones/pdf.png</normaloff>:/Icones/Icones/pdf.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stat">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>620</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(58, 195, 197);
    color: white;
    border-radius: 5px;
    padding: 5px;
}
</string>
      </property>
      <property name="text">
       <string>Statistiques</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxTrie">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>90</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Type</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="tab_etudiant">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>561</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>561</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="showMessagePushButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>notification</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1211</width>
       <height>801</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Icones/Icones/background.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_20">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(97, 208, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/img/menu.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>71</width>
       <height>701</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: rgb(178, 225, 229);</string>
     </property>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/home.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/calendrier.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>430</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/notepad.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/notification.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lecture.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_21">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>540</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/chat.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>640</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/togglapp.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>461</width>
       <height>681</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    font: 12px;
 background-color:transparent;
}

QLineEdit {
    border: 2px solid gray;
    border-radius: 5px;
    padding: 5px;
}

QPushButton {
    background-color: rgb(58, 195, 197);
    color: white;
    border-radius: 5px;
    padding: 5px;
}

QLineEdit:focus, QPushButton:focus {
    border: 2px solid #4CAF50;
}

QLabel {
    font-weight: bold;
    padding: 1px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="Ajouter">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
   color:#ffb001;
}</string>
      </property>
      <property name="text">
       <string>Equipement</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="nom_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nam">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="organization_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_Type">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="budget_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prix">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>360</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Icones/Icones/ajouter.png</normaloff>:/Icones/Icones/ajouter.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>450</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Icones/Icones/supprimer.png</normaloff>:/Icones/Icones/supprimer.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>400</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Icones/Icones/modifier.png</normaloff>:/Icones/Icones/modifier.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="id_label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>quantite</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_quantite">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>330</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_find">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>400</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="date_label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>id </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_IDS">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>450</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="date_label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>id </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>490</y>
        <width>321</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string>Qr code</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qr_code">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>580</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
    background-color: rgb(58, 195, 197);
    color: white;
    border-radius: 5px;
    padding: 5px;
}

QLineEdit:focus, QPushButton:focus {
    border: 2px solid #4CAF50;
}

</string>
      </property>
      <property name="text">
       <string>Qr_code</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Icones/Icones/chat.png</normaloff>:/Icones/Icones/chat.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </widget>
    <zorder>label</zorder>
    <zorder>afficher</zorder>
    <zorder>nom_gestion</zorder>
    <zorder>pushButton_20</zorder>
    <zorder>widget</zorder>
    <zorder>groupBox_2</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1236</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
